### PR TITLE
feat: parse all id token claims into raw_claims, so they are availabl…

### PR DESCRIPTION
**Pull request by pr1ze**
…e in the jsonnet context during registration. Fixing issue #2528 https://github.com/ory/kratos/issues/2528

Docs for https://github.com/ory/kratos/pull/2765 which should be merged and released first

## Related Issue or Design Document
[#2528](https://github.com/ory/kratos/issues/2528)

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [x] I have referenced an issue containing the design document if my change introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added the necessary documentation within the code base (if appropriate).

## Further comments

Docs for https://github.com/ory/kratos/pull/2765 which should be merged and released first

### CODE REVIEW

File: docs/kratos/social-signin/90_data-mapping.mdx [hunk 1]
```diff
@@ -141,28 +141,47 @@ The `std.ExtVar('claims')` object has the following structure and keys available
...
+	Gender              string                  `json:"gender,omitempty"`
+	Birthdate           string                  `json:"birthdate,omitempty"`
+	Zoneinfo            string                  `json:"zoneinfo,omitempty"`
+	Locale              string                  `json:"locale,omitempty"`
+	PhoneNumber         string                  `json:"phone_number,omitempty"`
+	PhoneNumberVerified bool                    `json:"phone_number_verified,omitempty"`
+	UpdatedAt           int64                   `json:"updated_at,omitempty"`
+	HD                  string                  `json:"hd,omitempty"`
+	Team                string                  `json:"team,omitempty"`
+  RawClaims           map[string]interface{}  `json:"raw_claims,omitempty"`
```
Comment (jonas-jonas) [inline docs/kratos/social-signin/90_data-mapping.mdx:167]:
```suggestion
    RawClaims           map[string]interface{}  `json:"raw_claims,omitempty"`
```

The formatting is off here.

File: docs/kratos/social-signin/90_data-mapping.mdx [hunk 2]
```diff
@@ -141,28 +141,47 @@ The `std.ExtVar('claims')` object has the following structure and keys available
...
+}
+```
+
+#### Raw Claims `raw_claims`
+
+All claims that are not part of the standard userinfo claims, are mapped into raw_claims. An example of mapping a Raw Claim called
```
Comment (jonas-jonas) [inline docs/kratos/social-signin/90_data-mapping.mdx:173]:
```suggestion
All claims that are not part of the standard userinfo claims, are mapped into `raw_claims`. An example of mapping a raw claim called
```

File: docs/kratos/social-signin/90_data-mapping.mdx [hunk 3]
```diff
@@ -141,28 +141,47 @@ The `std.ExtVar('claims')` object has the following structure and keys available
...
+groups:
```
Comment (jonas-jonas) [inline docs/kratos/social-signin/90_data-mapping.mdx:174]:
```suggestion
"groups":
```

**Review by jonas-jonas — CHANGES_REQUESTED** [review]
Just a few formatting & mark-up issues. Otherwise, looking really solid! Thanks!

**Comment by aeneasr** [comment]
I tried pushing some changes required for merging the PR to your fork & branch, but it appears that I am not allowed to do so 😕 

```
% git push ...
ERROR: Permission to push denied to aeneasr.
fatal: could not read from the remote repository.

Please make sure that you have the correct access rights
and the repository exists.
```

But the good news is, giving access is easy! :relaxed: All you need to do is [enable write access for maintainers](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork). Thank you! 😄

If the repository belongs to an organization, please add me for the [project as a collaborator](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/adding-outside-collaborators-to-repositories-in-your-organization)!

**Comment by pr1ze** [comment]
We have fixed the permissions issues now @aeneasr :) It looks like the PR is closed without being merged?

**Comment by aeneasr** [comment]
Yes, but the changes are on master. I pushed my changes to a different branch / PR :)

https://github.com/ory/docs/commit/fcbb06eba40c6d45ae932b019a21eb4688c3b219

**Pull request closed without merging**